src/labels/CustomLabels.labels

```diff
@@ -723,6 +723,14 @@
         <shortDescription>Context for when to add a date, amount, or detail field to a Rollup__mdt record.</shortDescription>
         <value>These fields determine what information is being summarized and provide the normal defaults for Donations. You can edit them as needed.</value>
     </labels>
+    <labels>
+        <fullName>CRLP_BatchSkipped</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error logged when a scheduled batch is skipped b/c yesterday's still running.</shortDescription>
```

Comment:
I think the the single quote needs to be escaped. If you refresh the label metadata file from your org it should pull down the properly formatted text for this label.